securedrop-keyring/debian/preinst

```diff
@@ -20,11 +21,11 @@ if [ -e /etc/apt/trusted.gpg ] && which gpg >/dev/null; then
 
    if gpg --homedir="$h" \
           --batch --no-tty --no-default-keyring --keyring /etc/apt/trusted.gpg \
-          --list-key 0x22245C81E3BAEB4138B36061310F561200F4AD77 > /dev/null 2>&1 ; then
+          --list-key 0x2359E6538C0613E652955E6C188EDD3B7B22E6A3 > /dev/null 2>&1 ; then
```

Comment:
yup, the one we want to remove is the new key that gets installed by the salt provisioning logic. it would be nice to remove this code, but until we stop writing the key to the general apt keyring via that salt logic and we know all pilot participants have updated after releasing that fix, we'll have to keep it in. sad, deleting code is so much fun.